lib/fathead/less/output.txt

```diff
@@ -0,0 +1,84 @@
e]</code> pair: first color and its relative position (position is optional)</li><li><code>color percent</code> pair: (optional) second color and its relative position</li><li>...</li><li><code>color percent</code> pair: (optional) n-th color and its relative position</li><li><code>color [percentage]</code> pair: last color and its relative position (position is optional)</li></ul><span class="prog__sub">Returns</span><p> <code>url</code> with &quot;URI-Encoded&quot; svg gradient.</p><span class="prog__sub">Example - colors stops in list</span><p></p><pre><code class="lang-less">div {\n  @list: red, green 30%, blue;\n  background-image: svg-gradient(to right, @list);\n}</code></pre><span class="prog__sub">equivalent - color stops in arguments</span><p></p><pre><code class="lang-less">div {\n  background-image: svg-gradient(to right, red, green 30%, blue);\n}</code></pre><span class="prog__sub">both result in</span><p></p><pre><code class="lang-css">div {\n  background-image: url(&apos;data:image/svg+xml,%3C%3Fxml%20version%3D%221.0%22%20%3F%3E%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20version%3D%221.1%22%20width%3D%22100%25%22%20height%3D%22100%25%22%20viewBox%3D%220%200%201%201%22%20preserveAspectRatio%3D%22none%22%3E%3ClinearGradient%20id%3D%22gradient%22%20gradientUnits%3D%22userSpaceOnUse%22%20x1%3D%220%25%22%20y1%3D%220%25%22%20x2%3D%22100%25%22%20y2%3D%220%25%22%3E%3Cstop%20offset%3D%220%25%22%20stop-color%3D%22%23ff0000%22%2F%3E%3Cstop%20offset%3D%2230%25%22%20stop-color%3D%22%23008000%22%2F%3E%3Cstop%20offset%3D%22100%25%22%20stop-color%3D%22%230000ff%22%2F%3E%3C%2FlinearGradient%3E%3Crect%20x%3D%220%22%20y%3D%220%22%20width%3D%221%22%20height%3D%221%22%20fill%3D%22url(%23gradient)%22%20%2F%3E%3C%2Fsvg%3E&apos;);\n}</code></pre><span class="prog__sub">Note</span><p> in versions before 2.2.0 the result is <code>base64</code> encoded .</p><span class="prog__sub">Generated background image has red color on the left, green at 30% of its width and ends with a blue color. Base64 encoded part contains following svg-gradient</span><p></p><pre><code class="lang-xml">&lt;?xml version=&quot;1.0&quot; ?&gt;\n&lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; version=&quot;1.1&quot; width=&quot;100%&quot; height=&quot;100%&quot; viewBox=&quot;0 0 1 1&quot; preserveAspectRatio=&quot;none&quot;&gt;\n    &lt;linearGradient id=&quot;gradient&quot; gradientUnits=&quot;userSpaceOnUse&quot; x1=&quot;0%&quot; y1=&quot;0%&quot; x2=&quot;100%&quot; y2=&quot;0%&quot;&gt;\n        &lt;stop offset=&quot;0%&quot; stop-color=&quot;#ff0000&quot;/&gt;\n        &lt;stop offset=&quot;30%&quot; stop-color=&quot;#008000&quot;/&gt;\n        &lt;stop offset=&quot;100%&quot; stop-color=&quot;#0000ff&quot;/&gt;\n    &lt;/linearGradient&gt;\n    &lt;rect x=&quot;0&quot; y=&quot;0&quot; width=&quot;1&quot; height=&quot;1&quot; fill=&quot;url(#gradient)&quot; /&gt;\n&lt;/svg&gt;</code></pre></section>	http://lesscss.org/functions/#misc-functions-svg-gradient
```

Comment:
Whoops looks like the SVG is wrapped in a `<pre><code>` block -- I don't think we want that